bin/faucet/src/handlers.rs

```diff
@@ -6,12 +6,18 @@ use miden_objects::{
     accounts::AccountId,
     assets::FungibleAsset,
     notes::{NoteId, NoteType},
+    transaction::OutputNote,
     utils::serde::Serializable,
 };
 use serde::{Deserialize, Serialize};
 use tracing::info;
 
-use crate::{errors::FaucetError, utils::FaucetState};
+use crate::{
+    errors::FaucetError,
+    utils::{build_client, FaucetState},
+};
+
+const TOKEN_AMOUNT_OPTIONS: [u64; 3] = [100, 500, 1000];
```

Comment:
wasn't this already part of the config? I can still see it in `miden-faucet.toml`